tests/Unit/PatchlevelEventSourcingBundleTest.php

```diff
@@ -46,21 +55,41 @@ public function testEnabled(): void
             ]
         );
 
+        self::assertTrue($container->has(DefaultController::class));
+        self::assertTrue($container->has(EventController::class));
+        self::assertTrue($container->has(InspectionController::class));
+        self::assertTrue($container->has(StoreController::class));
+        self::assertTrue($container->has(SubscriptionController::class));
 
-        self::assertEquals(DefaultController::class, DefaultController::class);
-        self::assertEquals(ProjectionController::class, ProjectionController::class);
-        self::assertEquals(StoreController::class, StoreController::class);
+        /** @messageHeaderRegistry MessageHeaderRegistry $messageHeaderRegistry */
+        $messageHeaderRegistry = $container->get(MessageHeaderRegistry::class);
+        self::assertTrue($messageHeaderRegistry->hasHeaderClass(RequestIdHeader::class));
     }
 
     private function compileContainer(ContainerBuilder $container, array $config): void
     {
+        $bundle = new PatchlevelEventSourcingBundle();
+        $bundle->build($container);
+
         $bundle = new PatchlevelEventSourcingAdminBundle();
         $bundle->build($container);
 
         $container->setParameter('kernel.project_dir', __DIR__);
 
         // services
 
+        $extension = new PatchlevelEventSourcingExtension();
+        $extension->load(
+            [
+                'patchlevel_event_sourcing' => [
+                    'connection' => [
+                        'url' => 'sqlite3:///:memory:',
+                    ],
+                ]
+            ],
+            $container
+        );
+
```

Comment:
I dislike this part here. I'm not sure though how to solve this in a better way.